src/App.js

```diff
@@ -0,0 +1,22 @@
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import Rockets from './pages/Rockets';
+import Mission from './pages/Missions';
+import Profile from './pages/Profile';
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar/>
+      <div>
+        <Routes>
+          <Route path='/' element={<Rockets />}></Route>
+          <Route path='missions' element={<Mission />}></Route>
+          <Route path='profile' element={<Profile />}></Route>
+        </Routes>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] Great Job So far your app looks awesome 👏 , but as part of the requirements the data fetching should occur only once for both: Rockets & Mission , that means once the page is rendered the app should do any fetching from the API on every time i navigate between pages because this will cause the redux state to be reseted every time i switch page, kindly make sure all the data is fetched once 🙏 

Hint : You can use UseEffect/UseMemo and check if the state is empty before sending API calls 

the error : 
![errorrs](https://github.com/ernestkorir/React-group-project/assets/89282221/83decb1c-830f-47d6-aba8-e6cf66393461)